data/text/common_1.asm

```diff
@@ -258,6 +259,7 @@ UnknownText_0x1bc44c::
 	line "ID no."
 	done
 
+; last Unused Mobile Adapter
```

Comment:
I'd put this as its own line, surrounded by two blank ones, after this text. `end of unused Mobile Text`. Surround the starting comment with blank lines as well, mostly for consistency with similar comments further down this file.